middlewares/middlewares.go

```diff
@@ -121,10 +146,18 @@ func JwtMiddleware(key string, algo string) negroni.Handler {
 			next(w, r)
 			return
 		}
-		err = jwtMiddleware.CheckJWT(w, r)
-		if err != nil {
-			log.Println("check jwt error", err.Error())
-			w.Write([]byte(fmt.Sprintf(`{"error": "%v"}`, err.Error())))
+
+		encounteredError := true
+		var handler http.HandlerFunc = func(w http.ResponseWriter, r *http.Request) {
+			encounteredError = false
+			user := r.Context().Value(jwtmiddleware.ContextKey{})
+			fmt.Println("user:", user)
+		}
+		middleware.CheckJWT(handler).ServeHTTP(w, r)
+
+		if encounteredError {
+			log.Println("check jwt error")
+			w.Write([]byte(`{"error": "Failed to validate JWT"}`))
 			return
 		}
 		next(w, r)
```

Comment:
You could replace this whole block simply with:

```
middleware.CheckJWT(next)
```

And not overriding the DefaultErrorHandler in the middleware.

Reply:
@sergiughf I am "struggling" to make this migration

can you help us with this migration?

Reply:
Sure thing:) how can I help?